pages/article/[articleLink].js

```diff
@@ -86,10 +91,10 @@ const Article = () => {
                 )}
               </figure>
             )}
-            {details && (
+            {details.length > 0 ? (
```

Comment:
Consider doing `details.length > 0 && <Component ...`  and dropping the string part off